README.md

```diff
@@ -1,43 +1,185 @@
 # Pingh
 
-An app notifying of new GitHub `@mention`s.
+A [GitHub App](https://github.com/apps/pingh-tracker-of-github-mentions) that notifies users 
+of new GitHub `@mention`s. It also runs as a system tray application on macOS.
 
-## Status
+- Displays a list of recent `@mention`s for a logged-in GitHub user.
+- Marks mentions as “read” or snooze them for later.
+- Sends notifications for new mentions and for mentions whose snooze time has expired.
+- Enters DND mode to turn off notifications.
 
-Currently, this is a work-in-progress example.
+## Tech
 
-## Planned Features
+The application consists of two parts: the server and the client, 
```

Comment:
Instead this whole thing:

```
Server stack:
-
-
-

Client stack:
-
-
-
```

And let's have a sub-section telling what to install for the local development and how.